app/backend/src/couchers/servicers/references.py

```diff
@@ -252,7 +272,9 @@ def ListPendingReferencesToWrite(self, request, context):
         with session_scope() as session:
             q1 = (
                 session.query(literal(True), HostRequest)
-                .outerjoin(Reference, Reference.host_request_id == HostRequest.conversation_id)
+                .outerjoin(Reference, HostRequest.conversation_id == Reference.host_request_id)
```

Comment:
Why did you swap the order? I prefer to have it `.join(Table, Table.col == Table2.col2)`, with the `Table` on "that" side of the join condition?